src/main/webapp/i18n/de/error.json

```diff
@@ -31,8 +29,8 @@
         "quizAlreadyOpenForPractice": "Das Quiz ist bereits zur Übung geöffnet.",
         "unknownAction": "Diese Aktion kann nicht durchgeführt werden.",
         "groupNotFound": "Die von dir eingegebene Gruppe wurde nicht gefunden.",
-        "noPrivacyStatementFile": "Es konnte keine Datenschutzerklärung gefunden werden.",
-        "emptyPrivacyStatement": "Die Datenschutzerklärung darf nicht leer sein.",
+        "noLegalDocumentFile": "Das angeforderte Dokument konnte nicht gefunden werden.",
+        "emptyLegalDocument": "Das Impressum oder die Datenschutzerklärung darf nicht leer sein.",
```

Comment:
Maybe have 2 messages for this? Optional IMO

Reply:
leave it like this for now as I think it's pretty self-explanatory which file could not be found if I navigated to the respective page